docs/adr/README.md

```diff
@@ -0,0 +1 @@
+# Architectural Decision Records
```

Comment:
Good start. I would add a list of records in this file so that links point to the appropriate doc.